apps_external_dns.tf

```diff
@@ -38,7 +38,7 @@ resource "helm_release" "external_dns" {
   name       = "external-dns"
   repository = "https://charts.bitnami.com/bitnami"
   chart      = "external-dns"
-  version    = "6.2.1"
+  version    = "6.3.1"
```

Comment:
There were some issues with `external-dns` in the past that were worked–around by picking some specific version here. Were they all fixed in the meantime? (sorry but I don't remember details :slightly_frowning_face:)

Reply:
I also only remember that vaguely - I deployed it on test env and everything _seems_ to work fine.